app/Http/Controllers/BaseController.php

```diff
@@ -30,8 +32,9 @@ class BaseController extends Controller
 
     public function index(): View
     {
+        $user=Auth::user();
         $model = new $this->model;
-
+        WelcomeEmailJob::dispatch($user);
```

Comment:
a little extra  code but a bit faster application

Reply:
Is there a  A simpler way

Reply:
no, your code looks very neat  and also you make it faster